src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: left 19px top 32px;
+
+  &__title {
+    margin-top: 206px;
+    margin-bottom: 4px;
+    font-family: $main-font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
+    padding-inline: $default-padding-inline;
+  }
+
+  &__product-code {
+    font-family: $main-font-family;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+    margin: inherit;
+    margin-bottom: 16px;
+    padding-inline: $default-padding-inline;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    padding-inline: $default-padding-inline;
```

Comment:
Ensure that the SCSS variable `$default-padding-inline` is defined in your SCSS files. Without this definition, the SCSS will not compile correctly.